src/main/scala/beam/router/R5RoutingApp.scala

```diff
@@ -0,0 +1,184 @@
+  implicit val householdAttributesFormat = new Format[HouseholdAttributes]
+  implicit val attributesOfIndividualFormat = new Format[AttributesOfIndividual]
+  implicit val intermodalUseFormat = new Format[IntermodalUse]
+  implicit val routingRequestFormat = new Format[RoutingRequest]
+
+  implicit val transitStopsInfoFormat = new Format[TransitStopsInfo]
+  implicit val beamPathFormat = new Format[BeamPath]
+  implicit val beamLegFormat = new Format[BeamLeg]
+  implicit val embodiedBeamLegFormat = new Format[EmbodiedBeamLeg]
+  implicit val embodiedBeamTripFormat = new Format[EmbodiedBeamTrip]
+  implicit val routingResponseFormat = new Format[RoutingResponse]
+}
+
+class RoutingHandler(val workerRouter: ActorRef) extends FailFastCirceSupport {
+  implicit val timeout: Timeout = new Timeout(10, TimeUnit.SECONDS)
+  import AllNeededFormats._
+
+  val route: Route = {
+    path("find-route") {
+      post {
+        entity(as[RoutingRequest]) { request =>
+          complete(workerRouter.ask(request).mapTo[RoutingResponse])
+        }
+      }
+    }
+  }
+}
+
+object CustomExceptionHandling extends LazyLogging {
+
+  def handler: ExceptionHandler = ExceptionHandler {
+    case t: Throwable =>
+      extractClientIP { remoteAddress =>
+        extractRequest { request =>
+          val msg = s"Exception during processing $request from $remoteAddress: ${t.getMessage}"
+          logger.error(msg, t)
+          complete(HttpResponse(StatusCodes.InternalServerError, entity = msg))
+        }
+      }
+  }
+}
+
+object R5RoutingApp extends BeamHelper {
+  import AllNeededFormats._
+
+  implicit val actorSystem: ActorSystem = ActorSystem()
+  implicit val materializer: ActorMaterializer = ActorMaterializer()
+  implicit val timeout: Timeout = new Timeout(600, TimeUnit.SECONDS)
+
+  def main(args: Array[String]): Unit = {
+    val (arg, cfg) = prepareConfig(args, isConfigArgRequired = true)
+    val workerRouter: ActorRef = actorSystem.actorOf(Props(classOf[R5RoutingWorker], cfg), name = "workerRouter")
+    val f = Await.result(workerRouter ? Identify(0), Duration.Inf)
+    logger.info("R5RoutingWorker is initialized!")
+
+    val maxHour = TimeUnit.SECONDS.toHours(new TravelTimeCalculatorConfigGroup().getMaxTime).toInt
+    val warmStart = BeamWarmStart(BeamConfig(cfg), maxHour)
+    logger.info(s"warmStart isEnabled?: ${warmStart.isWarmMode}")
+
+    warmStart.read.foreach { travelTime =>
+      workerRouter ! UpdateTravelTimeLocal(travelTime)
+      logger.info("Send `UpdateTravelTimeLocal`")
+    }
+
+    val interface = "0.0.0.0"
+    val port = 9000
+    val routingHandler = new RoutingHandler(workerRouter)
+    val boostedRoute = handleExceptions(CustomExceptionHandling.handler)(routingHandler.route)
+    Http().bindAndHandle(boostedRoute, interface, port)
+    logger.info(s"Http server is ready and bound to $interface:$port")
+  }
+
+  def printRoutingRequestJsonExample(): Unit = {
```

Comment:
Was this for testing purposes and left in? I see no usage of it, so maybe it can be removed now?

Reply:
I found #1721 which I linked in the case (remove if wrong), but it still doesn't entirely answer the reason.

Reply:
Are you about `printRoutingRequestJsonExample`? I left it just in case if someone does not know how RoutingRequest looks as a JSON.

Reply:
I'm not sure how #1721 is connected to this PR, removed.